src/cambridge.json

```diff
@@ -0,0 +1,21 @@
+{
+  "publisher": "Cambridge Core",
+  "publisher_parent": "Cambridge University Press",
+  "publisher_member": null,
+  "crossref_member": null,
+  "prefixes": ["10.1017"],
+  "urls": {
+    "pdf": "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/%s.pdf"
```

Comment:
Can you give some examples that work?  This URL doesn't work at least for the few DOIs I tried., e.g, this https://www.cambridge.org/core/services/aop-cambridge-core/content/view/10.1017/s0031819114000357.pdf doesn't work . This URL https://www.cambridge.org/core/services/aop-cambridge-core/content/view/S0031819114000357 does work for that DOI, but it's paywalled so if you don't have access it won't work 

for this publisher seems like the pattern should work , BUT they do seem to be a mess, e.g., 

this DOI 10.1557/opl.2014.876 has a link that looks like https://www.cambridge.org/core/services/aop-cambridge-core/content/view/S1946427414008768 - so completely unrelated to its DOI for the most part at least

Reply:
I fixed the other issues you noted, but will have to address this in a bit...

Reply:
okay